src/components/App.jsx

```diff
@@ -19,40 +26,38 @@ const PaperView = props => {
 }
 
 const AppRouter = () => {
-  return (
-    <Routes>
+  const routes = (
+    <Route path="/" element={<AppLayout />}>
       <Route path="/paper/*" element={<PaperView />} />
       <Route path="*" element={<Navigate to="/paper" replace />} />
-    </Routes>
+    </Route>
   )
+  const router = createHashRouter(createRoutesFromElements(routes))
+  return <RouterProvider router={router} />
```

Comment:
What is the point of using new "Data APIs" instead of old-school `HashRouter` components? What does it bring? Is it equivalent to the old way or not? :thinking:

Reply:
A ma connaissance, pour notre utilisation, c'est sensiblement la même chose.
Mais je pense qu'il est bon de se familiariser avec les nouvelles approches (tant que ce n'est pas contre productif)